plugins/auth-backend/package.json

```diff
@@ -66,7 +66,7 @@
     "passport-github2": "^0.1.12",
     "passport-gitlab2": "^5.0.0",
     "passport-google-oauth20": "^2.0.0",
-    "passport-microsoft": "^0.1.0",
+    "passport-microsoft": "^1.0.0",
```

Comment:
Note: This is a major bump but contains no breaking changes afaics, it's a stable api major bump.

Reply:
Yep, seems fine to ship as a patch 👍